validators/kernel_validator_test.go

```diff
@@ -31,22 +31,42 @@ func TestValidateKernelVersion(t *testing.T) {
 	// they may be different.
 	// This is fine, because the test mainly tests the kernel version validation logic,
 	// not the DefaultSysSpec. The DefaultSysSpec should be tested with node e2e.
-	testRegex := []string{`^3\.[1-9][0-9].*$`, `^([4-9]|[1-9][0-9]+)\.([0-9]+)\.([0-9]+).*$`}
+	testRegex := []string{`^4\.4.*$`, `^4\.19.*$`, `^4\.[2-9][0-9].*$`, `^([5-9]|[1-9][0-9]+)\.([0-9]+)\.([0-9]+).*$`}
 	for _, test := range []struct {
 		name    string
 		version string
 		err     bool
 	}{
 		{
-			name:    "3.19.9-99-test first version regex matches",
+			name:    "3.19.9-99-test no version regex matches",
 			version: "3.19.9-99-test",
-			err:     false,
+			err:     true,
 		},
 		{
-			name:    "4.4.14+ one of version regexes matches",
+			name:    "4.4.14+ no version regex matches",
 			version: "4.4.14+",
 			err:     false,
 		},
+		{
+			name:    "4.17.3 no version regex matches",
+			version: "4.17.3",
+			err:     true,
+		},
```

Comment:
can you please also add a test case for 4.6 or something else that is between the LTS versions and returns an error?

Reply:
Added.